#205 - Adding Will Whang's RPI4 Clock (#218)
geerlingguy/raspberry-pi-pcie-devices: _boards_cm/will-whangs-rpi4-clock.md

---
layout: board
title: "Will Whang's RPi4 Clock"

> short_description: A beautifully compact CM4-based clock prototype built with the Symmetricom Rubidium Oscillator.

Reviewer: ```suggestion
short_description: A compact CM4 clock with GNSS integration and a rubidium atomic clock.
```

status: prototype
picture: "/images/board-will-whangs-rpi4-clock.jpg"
github_issue: "https://github.com/geerlingguy/raspberry-pi-pcie-devices/issues/205"
link: https://twitter.com/will_whang/status/1431004730973556739/photo/1
videos: []
---

> The RPI4 Clock by Will Whang ([@will127534](https://github.com/will127534)) uses the Symmetricom Rubidium Oscillator in a similar way to the [Time Card](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwi2qLagxuTyAhWKPM0KHdzCAFUQFnoECAMQAQ&url=https%3A%2F%2Fengineering.fb.com%2F2021%2F08%2F11%2Fopen-source%2Ftime-appliance%2F&usg=AOvVaw3Y_flunwDe2ZLdTI634iFh).

Reviewer: ```suggestion
The RPI4 Clock by Will Whang ([@will127534](https://github.com/will127534)) uses the Symmetricom Rubidium Oscillator in a similar way to the [Time Card](https://github.com/opencomputeproject/Time-Appliance-Project/tree/master/Time-Card#readme).
```
